app.js

```diff
@@ -0,0 +1,17 @@
+const express = require ('express')
+const app = express()
+const {getApi, getTopics} = require('./apiController')
+
+
+
+app.get("/api", getApi)
+
+app.get('/api/topics', getTopics)
+
+app.use((req, res, next) => {
```

Comment:
Which parameter would indicate that this is error handling middleware within your chain?